lms/templates/user_dropdown.html

```diff
@@ -0,0 +1,59 @@
+## mako
+<%page expression_filter="h"/>
+<%namespace name='static' file='static_content.html'/>
+
+<%!
+from django.core.urlresolvers import reverse
+from django.utils.translation import ugettext as _
+
+from openedx.core.djangoapps.user_api.accounts.image_helpers import get_profile_image_urls_for_user
+%>
+
+% if uses_pattern_library:
```

Comment:
I started to do this, but then realized that this would make it more likely that someone would fix one version but not the other. I started to add comments to each file referring to the other, but now I think I prefer keeping it all in one file so that it is obvious what needs to change.